notebooks/TestGrid_EDA.ipynb

```diff
@@ -89,6 +89,7 @@
        " '\"redhat-openshift-okd-release-4.6-blocking\"',\n",
```

Comment:
<p>Will it be a good idea if we create multiple columns from values column and name them according to time?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>I don't think that would be a good idea. Each grid has different timestamp values, so that would create grid dataframes that all  have different number of columns and different column names.  I think this approach is more organized and consistent. </p>

Reply:
makes sense :+1: